patches/net/minecraft/FileUtil.java.patch

```diff
@@ -27,7 +27,7 @@
 @@ -74,7 +_,7 @@
  
      public static boolean isPathPortable(Path p_133735_) {
-         for(Path path : p_133735_) {
+         for (Path path : p_133735_) {
 -            if (RESERVED_WINDOWS_FILENAMES.matcher(path.toString()).matches()) {
 +            if ((net.neoforged.neoforge.common.NeoForgeMod.getProperFilenameValidation() ? RESERVED_WINDOWS_FILENAMES_NEOFORGE : RESERVED_WINDOWS_FILENAMES).matcher(path.toString()).matches()) {
```

Comment:
Is this necessary for this change? It seems unrelated to me.

Reply:
Wondering if our IDEs keep adding the space. It is showing up for me now when I do unpack sources. My original pr to this area did not have the space
https://github.com/neoforged/NeoForge/pull/767/files

Reply:
Oh, I misread the diff, silly me. Yeah, it seems like this was an extraneous whitespace change, which we can ignore or just merge to stop it from happening again.